### DISCUSSION TITLE
Charm not finding T1 input file

### DISCUSSION THREAD

**Discussion by DarthTroopers** [simnibs/simnibs#338] [7.Creating head models]
Hello,

I'm pretty new to simnibs and am experiencing some difficulties with segmentation using charm. I am continuing to receive an error that the image file cannot be found even though I believe I have navigated to the correct directory.

![image](https://github.com/simnibs/simnibs/assets/169263846/7ac7b931-3573-4c66-b71a-445093af9812)

Hopefully the image shows up. My file explorer is shown below with the T1 and T2 files. Please let me know what I am doing wrong. Thank you!

![image](https://github.com/simnibs/simnibs/assets/169263846/2c151e65-1adc-4147-bc19-5e25c47eff63)

**Comment by oulap** [comment]
Hi,

I can see that in your terminal you cd'ed into the org folder, i.e. the terminal shows:

C:\Users\Owner\Downloads\simnibs4_examples\org

Note that in the charm command the image files are passed like this:

org/ernie_T1.nii.gz and org/ernie_T2.nii.gz

so charm tries to look for the org-folder and then the T1 and T2 files. Now because you're already inside the org-folder in the terminal charm can't find the files. You can do two things:

1) Go one folder up by calling: `cd ..` and then repeat the charm command.

2) call charm like this inside the org-folder: `charm ernie ernie_T1.nii.gz ernie_T2.nii.gz`

Hope that makes sense!

**Comment by DarthTroopers** [comment]
Hello,

Thank you for the help. I moved up a directory but am now getting an error that forcerun has be set to overwrite the m2m folder.

![image](https://github.com/simnibs/simnibs/assets/169263846/8c85e9e6-1e24-4dc2-8768-2f577167d649)

I deleted the folder that was generated from the earlier run. Perhaps it is conflicting with the m2m_ernie directory under simnibs4_examples? I should also not I removed the .gz as my files appear as .nii files.  Thank you

**Reply by DarthTroopers** [reply]
As I suspected that was the issue, I have now generated a segmentation. Thanks again.

**Reply by oulap** [reply]
Yes, simnibs does not overwrite results by default, so you need to either remove the folder, or use the --forcerun flag to tell simnibs that it's okay to overwrite.
